lib/quby/questionnaires/entities/flag.rb

```diff
@@ -18,6 +18,16 @@ def initialize(key:,
         def if_triggered_by(answer_flags)
           yield if answer_flags[key] == trigger_on
         end
+
+        def to_codebook(_options = {})
+          output = []
+          output << "#{key} flag"
+          output << " 'true' - #{description_true}"
+          output << " 'false' - #{description_false}"
+          output << " '' (leeg) - Vlag niet ingesteld, informatie onbekend"
```

Comment:
Volgens mij is dat niet nodig, de default is altijd het lege geval. De trigger_on heeft geen invloed op de beginwaarde.